dask/array/ufunc.py

```diff
@@ -12,9 +12,10 @@
 from ..highlevelgraph import HighLevelGraph
 from ..utils import skip_doctest, funcname
 try:
-    from ..dataframe import core as ddcore
+    from ..dataframe.utils import is_dataframe_like
 except ImportError:
-    pass
+    def is_dataframe_like(df):
+        return False
```

Comment:
We could also move the current `is_dataframe_like` function down to `dask/utils.py`.  I don't think it imports anything itself.

If we did this, we would want to maintain a reference to it in `dask/dataframe/utils.py` for backwards compatibility.

Reply:
We could do that as well. I have no strong opinions on doing this or keeping @jakirkham's previous solution. Thoughts, @jakirkham?

Reply:
Sorry I missed this. If you wanted to move things around, I don't really have any objections as noted earlier. Though I don't feel a strong need to do that either. Will defer to Matt here.

Reply:
This has since happened. So marking this as resolved. If there are additional changes needed w.r.t. how these have been moved, please let us know on the new diff. 🙂